internal/provider/vsphere/ignition.go

```diff
@@ -1,10 +1,50 @@
 package vsphere
 
-import "github.com/openshift/assisted-service/internal/common"
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/openshift/assisted-service/internal/common"
+)
 
 func (p vsphereProvider) PreCreateManifestsHook(cluster *common.Cluster, envVars *[]string, workDir string) error {
 	return nil
 }
-func (p vsphereProvider) PostCreateManifestsHook(cluster *common.Cluster, envVars *[]string, workDir string) error {
+func (p vsphereProvider) PostCreateManifestsHook(_ *common.Cluster, _ *[]string, workDir string) error {
+	// Deleting machines and machineSets for vsphere platform after manifest generation
+	// The following steps are included in the Openshift UPI vSphere installation guide. Go to step 2 in the link below:
+	// https://docs.openshift.com/container-platform/4.9/installing/installing_vsphere/installing-vsphere.html#installation-user-infra-generate-k8s-manifest-ignition_installing-vsphere
+
+	// Delete machines
+	p.Log.Info("Deleting machines manifests")
+	files, _ := filepath.Glob(path.Join(workDir, "openshift", "99_openshift-cluster-api_master-machines-*.yaml"))
```

Comment:
i suggest you do the Glob with priority number:
99_openshift-cluster-api_master-machines-*.yaml -> *_openshift-cluster-api_master-machines-*.yaml

Reply:
eliminating the '*'s? can you please send me a link to this pattern?

Reply:
you meant replace the priority number with "*"

Reply:
No, GitHub just swalloed the `*`. He meant:
`99_openshift-cluster-api_master-machines-*.yaml -> *_openshift-cluster-api_master-machines-*.yaml`